plugins/custom-dashboard/README.md

```diff
@@ -0,0 +1,39 @@
+# Dashboard Plugin
+
+A plugin that enables dashboards of user defined plots in the FiftyOne app.
+
+https://github.com/user-attachments/assets/2ec5946e-ac43-48fa-b0a9-4d1d545ba0fc
+
+## Installation
+
+```shell
+fiftyone plugins download \
+    https://github.com/voxel51/fiftyone-plugins \
+    --plugin-names @voxel51/dashboard-builder
```

Comment:
Needs to be updated to match the final plugin name, which I propose is `@voxel51/dashboard`